### DISCUSSION TITLE
Getting Ready for now env

### DISCUSSION THREAD

**Discussion by AndrewBarba** [vercel/vercel#4048] [General] [answered]
I've been closely following the development of [`now env`](https://github.com/zeit/now/pull/3999) and am getting anxious for additional docs on how it will work with domains and deployments.

This ultimately solves a ton of the issues people had with Now v17 and also solves the dual project structure a lot of us are using in order to get staging/prod environments.

Are there any preview docs we can read to get ready for the change? Any additional workflows this will help solve?

**Comment by styfle** [comment, marked as answer]
Yes! This feature has been released in [Now 18.0.0](https://github.com/zeit/now/releases/tag/now%4018.0.0) and is available in the dashboard today!
